ocelot/settings.py

```diff
@@ -166,6 +166,11 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```

Comment:
@snukky: again, should we switch to `pathlib.Path`?

Reply:
Sure. Will do.

Reply:
As mentioned in another pull request, I will open a new PR for this.